src/messaging/ReliableMessageMgr.cpp

```diff
@@ -416,6 +417,32 @@ void ReliableMessageMgr::RegisterSessionUpdateDelegate(SessionUpdateDelegate * s
     mSessionUpdateDelegate = sessionUpdateDelegate;
 }
 
+CHIP_ERROR ReliableMessageMgr::MapSendError(CHIP_ERROR error, uint16_t exchangeId, bool isInitiator)
+{
+    if (error == CHIP_ERROR_POSIX(ENOBUFS)
```

Comment:
This should not be looked-at for LWIP. I think we need to have less platform-specifics in here.

Please use #if and #else here

Reply:
I can do that.  I was a little surprised we even compiled CHIP_ERROR_POSIX bits on LWIP.....

Reply:
Done.